### ISSUE TITLE
Caching package index

### ISSUE THREAD

**Issue by lassepe** [fannheyward/coc-julia#228]
It seems that the extension has to index the packages of the environment at every startup; even if the the environment was already indexed before and no changes where made to the Manifeset/Project.jl. Is there a config-option to write the indexing results to disk? It seems like the vscode-extensions does something like that.

**Comment by fannheyward** [comment]
Can you share which vscode-julia configuration does this? And does LanguageServer.jl support this?

**Comment by lassepe** [comment]
I just inferred this from [this comment on discourse] (https://discourse.julialang.org/t/vs-code-julia-language-server-caching/38967/2). I haven't found the code that does this yet but I'm happy to dig.

**Comment by lassepe** [comment]
It seems like caching to disk already happens since the symbol store directory is passed to the language server instance [here](https://github.com/fannheyward/coc-julia/blob/c3c5362beafc1dc88cd9ec706c9d83b6de619a8b/server/JuliaLS/src/main.jl#L29) and I also found the disk cache at `~/.config/coc/extensions/coc-julia-data/symbolstore2/`.

Still, even with this disk cache in place I am seeing the message `Julia Language Server Indexing packages...` after the first time I use the "jump to definition" and this still takes about 10 seconds. While caching seems to speed things up (I deleted the disk cache and then indexing took very long), I am surprised that only loading the disk cache takes this long. In particular, since I'm already using the precompiled system image. Maybe the delay is just jit overhead because some of the indexing routines did not get hit by the workload in `precompile.jl`?
